src/htm/algorithms/SpatialPooler.hpp

```diff
@@ -287,7 +287,6 @@ class SpatialPooler : public Serializable
     ar(CEREAL_NVP(overlapDutyCycles_));
     ar(CEREAL_NVP(activeDutyCycles_));
     ar(CEREAL_NVP(minOverlapDutyCycles_));
-    ar(CEREAL_NVP(tieBreaker_));
```

Comment:
Since tieBreaker is deterministic it does not need to be saved so this is correct.  But are you sure it is regenerated after the restore?  If it should be different then answers will eventually drift off of the correct set on subsequent iterations.

Reply:
no, tie breakers are totally removed and not used, as they did not have 100% the effect we expected. And you've obsoleted them in the long-dragging Deterministic builds PR  :+1: where you figured we need to add the `a > b` to the segment ordering algorithm. 

This is because tie breakers sometimes work and break ties:
{1,1} + {0, 0.01} -> L < R

but sometimes they just break (we apply tie breaks flat out to all, for performance reasons)
{1, 0.99} + {0, 0.01} -> L ? R broken

TL;DR this PR removes tieBreakers altogether because those are not needed (we do deterministic sort)

Reply:
see the removal https://github.com/htm-community/htm.core/pull/534#discussion_r299778927

Reply:
Oh, somehow I missed the fact that they were removed.  I like that you did that.